tools/taskcluster/nimbledroid_upload.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script installs google cloud sdk, logs into google firebase, executes tests,
+# and collects test artifacts into the test_artifacts folder
+
+# If a command fails then do not proceed and fail this script too.
+set -e
```

Comment:
Please add a big warning comment here or below mentioning why we do not want to print those commands.

Also is it easy to revoke and create a new API key in case anything leaks?

And even better: Are there no other ways to supply the user to CURL that does not leak the key to the command line?